test/integration/model/find.test.js

```diff
@@ -121,7 +121,7 @@ describe(Support.getTestDialectTeaser('Model'), function() {
           where: ['specialkey = ?', 'awesome'],
           logging: function(sql) {
             test = true;
-            expect(sql).to.contain("WHERE specialkey = 'awesome'");
+            expect(sql).to.match(/WHERE specialkey = N?'awesome'/);
```

Comment:
Ah, that question mark added in the fix is just a regular expression quantifier, meaning one or zero N characters are expected (to cover cases for mssql - one - and other dialects - zero - respectively)